### DISCUSSION TITLE
Ant Media Muting RTMP Source

### DISCUSSION THREAD

**Discussion by hyalicom** [ant-media/Ant-Media-Server#5189] [Q&A]
Hi, I would like to ask, is there any way to stream with no audio? I have a live streaming source rtmp which i am streaming to social media platforms, i want to stream with no audio.. is that possible in antmedia?

**Comment by yashtandon113** [comment]
@hyalicom 
Yes, you can stream to the social media using Ant Media Server but as far as I know, platform like YouTube does not accept streams without audio track as per it's policy. I mean if there will be no audio track in your stream then it cannot be published on YouTube as AMS does not add any dummy track.

You can add dummy audio track to your stream using Ant Media Server Filter plugin. Please check the [**following discussion**](https://community.antmedia.io/topic/1418-cant-get-to-stream-to-youtube-from-ant-media-app-community-edition-in-linode).
